app/exchanges/run_csv/pacbio_sample_sheet_v13.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+# A class spefically for SMRT-Link v13 Sample Sheets, based on the PacbioSampleSheet class
+
+module RunCsv
+  # RunCsv::PacbioSampleSheetV13
+  # Used to generate sample sheets specific to the PacBio pipeline for v13
+  class PacbioSampleSheetV13 < PacbioSampleSheet
+    # Generate a hash of settings for the run
+    def run_settings
+      run = object # Pacbio::Run
+      first_plate = run.plates[0]
+      second_plate = run.plates[1]
+      {
+        'Instrument Type' => run.system_name, # Revio
+        'Run Name' =>	run.name, # TRACTION-RUN-1042
+        'Run Comments' =>	run.comments, # TRAC-2-7242 245pM TRAC-2-782 247pM
+        'Plate 1' =>	first_plate.sequencing_kit_box_barcode, # 1021188000328660070020240502
+        'Plate 2' =>	second_plate&.sequencing_kit_box_barcode, # 1021188000328660070020240503
+        'CSV Version' =>	1
+      }
+    end
+
+    # Generate a list of plate-well identifiers.
+    # Eg. ['1_A01', '1_A02', '2_A01', ...]
+    def plate_well_names
+      wells = object.plates.flat_map(&:wells)
+      wells.map do |well|
+        "#{well.plate.plate_number}_#{well.position_leading_zero}"
+      end
+    end
+
+    # Generate a hash of settings for a single cell
+    def generate_smrt_cell_settings(well) # rubocop:disable Metrics/MethodLength
+      {
+        'Well Name'	=> well.pools.first.tube.barcode, # TRAC-2-7242
```

Comment:
What if the well has a library? Might be worth checking with Ben on the best way forward.